pylint/checkers/base.py

```diff
@@ -53,12 +54,12 @@ class NamingStyle(object):
     # has multiple "accepted" forms of regular expressions,
     # but we need to special-case stuff like dunder names
     # in method names.
-    CLASS_NAME_RGX = None
-    MOD_NAME_RGX = None
-    CONST_NAME_RGX = None
-    COMP_VAR_RGX = None
-    DEFAULT_NAME_RGX = None
-    CLASS_ATTRIBUTE_RGX = None
+    CLASS_NAME_RGX = Pattern[str]
```

Comment:
Does this class behave like `None` when we define a variable with it?

Reply:
will check for it.

Reply:
should these instead be `CLASS_NAME_RGX: Pattern[str]`

Reply:
Unfortunately that's 3.6 syntax only so won't work for us just yet (we still support 3.4 and 3.5)